src/main/java/org/broadinstitute/hellbender/utils/read/SAMRecordToGATKReadAdapter.java

```diff
@@ -447,6 +447,13 @@ public void setIsUnplaced() {
         samRecord.setMappingQuality(SAMRecord.NO_MAPPING_QUALITY);
     }
 
+    @Override
+    public void setInferredInsertSize(int insertSize) {
```

Comment:
@jamesemery I missed this somehow before, but this exists already as setFragmentLength